Data_Structures/AVL_tree/AVL.java

```diff
@@ -0,0 +1,315 @@
+package Data_Structures.AVL_tree;
+
+import java.util.Scanner;
+
+
```

Comment:
Why do we have extra line changes here?

Reply:
Okay Sir, I have removed the unnecessary spaces.